examples/abitest/config/config.toml

```diff
@@ -0,0 +1,9 @@
+name = "abitest"
+
+[modules]
+frontend_module = { path = "examples/target/wasm32-unknown-unknown/release/abitest_0_frontend.wasm" }
+backend_module = { path = "examples/target/wasm32-unknown-unknown/release/abitest_1_backend.wasm" }
+
+[initial_node_configuration]
+wasm_module_name = "app"
```

Comment:
I'm confused, shouldn't this value be "frontend_module"?

Reply:
Done